src/test/java/com/akademia/educators_back/service/impl/CommentServiceImplTestVerify.java

```diff
@@ -0,0 +1,179 @@
+package com.akademia.educators_back.service.impl;
+
+import com.akademia.educators_back.entity.CommentEntity;
+import com.akademia.educators_back.entity.ProblemEntity;
+import com.akademia.educators_back.entity.UserEntity;
+import com.akademia.educators_back.mapper.CommentMapper;
+import com.akademia.educators_back.repository.CommentRepository;
+import com.akademia.educators_back.to.CommentTo;
+import com.akademia.educators_back.validator.CommentValidator;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.boot.test.context.SpringBootTest;
+
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.mockito.Mockito.*;
+
+@SpringBootTest
+@ExtendWith(MockitoExtension.class)
+public class CommentServiceImplTestVerify {
```

Comment:
w klasach i metodach testowych nie piszemy słówek public, różnego rodzaju quality checkery to zawsze podkreślają